src/components/instantlaunches/listing/index.js

```diff
@@ -26,12 +26,24 @@ import {
     LIST_INSTANT_LAUNCHES_BY_METADATA_KEY,
     listInstantLaunchesByMetadata,
 } from "serviceFacades/instantlaunches";
+import {
+    getResourceUsageSummary,
+    RESOURCE_USAGE_QUERY_KEY,
+} from "serviceFacades/dashboard";
+import { getUserQuota } from "common/resourceUsage";
+import { useConfig } from "contexts/config";
+import globalConstants from "../../../constants";
+import withErrorAnnouncer from "components/error/withErrorAnnouncer";
 
 function Listing(props) {
-    const { baseId } = props;
+    const { baseId, showErrorAnnouncer } = props;
     const [showAppDetails, setShowAppDetails] = useState(null);
+    const [config] = useConfig();
+    const [computeLimitExceeded, setComputeLimitExceeded] = useState(
+        !!config?.subscriptions?.enforce
+    );
 
-    const { t } = useTranslation("instantlaunches");
+    const { t } = useTranslation("instantlaunches", "dashboard");
```

Comment:
This should be an array:
```suggestion
    const { t } = useTranslation(["instantlaunches", "dashboard"]);
```

Reply:
🤦‍♀️ I should have caught that. Thanks for catching it.